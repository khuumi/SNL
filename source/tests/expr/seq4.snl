[47, "jalepeno", -0.482]