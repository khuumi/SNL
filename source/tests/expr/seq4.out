List [(Int 47), (String jalepeno), (Unop Negate (Float 0.482))]